packages/create-remix/templates/deno/server.js

```diff
@@ -0,0 +1,13 @@
+import { serve } from "https://deno.land/std/http/server.ts";
+
+import { createRequestHandlerWithStaticFiles } from "@remix-run/deno";
+import * as build from "@remix-run/dev/server-build";
+
+const remixHandler = createRequestHandlerWithStaticFiles({
+  build,
+  mode: process.env.NODE_ENV
```

Comment:
Let's add a global variable that our compiler knows how to substitute during the build called `REMIX_MODE`.

Reply:
Actually, let's just put a TODO on this line and add this var in a separate PR.